webdriver/transport.py

```diff
@@ -43,7 +43,7 @@ def from_http_response(http_response):
             #
             # > 4. If data is not null, let response's body be a JSON Object
             #      with a key `value` set to the JSON Serialization of data.
-            assert "value" in body
+            assert "value" in body or "error" in body
```

Comment:
Potentially with an assertion message like, "response body specifies exactly one of 'value' or 'error'" since the expression on its own is admittedly a little hard to understand.